native/dlartg.go

```diff
@@ -17,7 +17,8 @@ import "math"
 func (impl Implementation) Dlartg(f, g float64) (cs, sn, r float64) {
 	safmin := dlamchS
 	eps := dlamchE
-	safmn2 := math.Pow(float64(dlamchB), math.Log(safmin/eps))
+	exp := int(math.Log(safmin/eps) / math.Log(float64(dlamchB)) / 2)
+	safmn2 := math.Pow(float64(dlamchB), float64(exp))
```

Comment:
>`dlamchB` is an int variable but yes, it would be nice if it were a const.

I've fixed that in #258. If you allow me to merge that, the expression above will work.